images/karpenter/README.md

```diff
@@ -0,0 +1,26 @@
+<!--monopod:start-->
+# karpenter
+| | |
+| - | - |
+| **Status** | experimental |
+| **OCI Reference** | `cgr.dev/chainguard/karpenter` |
+| **Variants/Tags** | ![](https://storage.googleapis.com/chainguard-images-build-outputs/summary/karpenter.svg) |
+
+*[Contact Chainguard](https://www.chainguard.dev/chainguard-images) for enterprise support, SLAs, and access to older tags.*
+
+---
+<!--monopod:end-->
+
+Minimal image with Karpenter. **EXPERIMENTAL**
+
+## Get It!
+
+The image is available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/karpenter:latest
+```
+
+## Using Karpenter
+
+The Chainguard Karpenter image contains the `karpenter` controller and is a drop-in replacement for the upstream image.
```

Comment:
Looks like you should be able to follow the [official installation instructions](https://karpenter.sh/v0.27.0/getting-started/getting-started-with-eksctl/) and pass this to helm to use our image:

```
--set controller.image=cgr.dev/chainguard/karpenter:latest
```